react-native/react/menubar/index.render.desktop.js

```diff
@@ -9,10 +9,12 @@ import {intersperse} from '../util/arrays'
 import {parseFolderNameToUsers, canonicalizeUsernames, stripPublicTag} from '../util/kbfs'
 
 import {globalStyles, globalColors} from '../styles/style-guide'
-import {Text, Input} from '../common-adapters/index.desktop.js'
+import {Text, Input, Terminal, Icon} from '../common-adapters/index.desktop.js'
 
 import {CircularProgress} from 'material-ui'
-import {cleanup} from '../util/kbfs'
+import {cleanup, allowLoggedOut as allowLoggedOutKBFS} from '../util/kbfs'
+// TODO use this instead of notification after merging
+// import {NotifyPopup} from '../native/notifications'
```

Comment:
Once this is merged back into the 'new menu' PR, i'll rebase that against master then fix this up